packages/happn-cluster/lib/services/orchestrator/serviceEntry.js

```diff
@@ -0,0 +1,110 @@
+//____________________________ SERVICE-REGISTRY ENTRY_____________
```

Comment:
inconsistent name, why serviceEntry.js? surely the name of this file should be service-entry.js

Reply:
changed